### DISCUSSION TITLE
Extension in migration name

### DISCUSSION THREAD

**Discussion by Fjandin** [sequelize/umzug#447] [Q&A]
Is there any way of ignoring extension name in the migration naming?
Problem is that i'm migrating to typescript so all migration files are now with `.ts`. When i execute my code with `ts-node` it will now see all migrations as not executed.

Is there any way of overriding the migration name or removing the extension part from the name?

**Comment by mmkal** [comment]
Are you using v2 or v3-beta? If v3 beta the `resolve` function might help with this. There are a couple of examples along these lines in the readme: https://github.com/sequelize/umzug#migration-files

**Reply by Fjandin** [reply]
Want to stick with v2 since it's used in production i'd rather wait for a final version, but i'll look into it. Thanks :)
